include/xcanvas/xcanvas_config.hpp

```diff
@@ -35,7 +35,7 @@
 // Semver requirement for jupyter-canvas
 #define XJUPYTER_CANVAS_VERSION_MAJOR 0
 #define XJUPYTER_CANVAS_VERSION_MINOR 11
-#define XJUPYTER_CANVAS_VERSION_PATCH 1
```

Comment:
Is this needed?

Reply:
oh das was not on purpose...not sure how and why I got that change ...good catch